### PR TITLE
IWE-56 BE: Remove existence of interest, including events, controller functions, methods, migrations, and models / relationships.

### PR DESCRIPTION
**Pull request by arthurwirsching**
### `IWE-56` - `PR`
`**READY**`

### Migrations
`YES`

### What's this PR do?
`Removed interests from application, can no longer interact with interests (apart from possibly the search functionality for now). `
### Where should the reviewer start?  
`Log in and check dashboard, make sure no more options to add interests are there. Make sure most of the site isn't more broken than it should be.`

### CODE REVIEW

File: app/Http/Controllers/PersonController.php [hunk 1]
```diff
@@ -1,47 +1,23 @@
...
-use Helix\Models\Research;
-use Helix\Models\Academic;
-use Helix\Models\Personal;
-use Helix\Models\Purpose;
-use Helix\Models\Attribute;
-use Helix\Events\Individual\IndividualAddAcademicInterests;
-use Helix\Events\Individual\IndividualAddPersonalInterests;
-use Helix\Events\Individual\IndividualAddInterests;
-
-use Illuminate\Database\Eloquent\ModelNotFoundException;
```
Comment (alejandrom2) [inline app/Http/Controllers/PersonController.php]: ?

File: app/Http/Controllers/PersonController.php [hunk 2]
```diff
@@ -1,47 +1,23 @@
...
 use Illuminate\Support\Collection;
-use Searchy;
```
Comment (alejandrom2) [inline app/Http/Controllers/PersonController.php]: ?

File: app/Http/Controllers/ProjectController.php [hunk 3]
```diff
@@ -276,17 +278,11 @@ public function getStep2($projectId = null)
 
         // TODO: refactor everything to the back end
 
-        if (request()->wantsJson()) {
-            if (\array_key_exists('interests', session('new-project'))) {
-                return jsonEncodeTags(session('new-project')['interests']['tags']);
-            }
-        }
 
         $categories = Research::whereNull('parent_attribute_id')->pluck('title', 'attribute_id as id');
```
Comment (alejandrom2) [inline app/Http/Controllers/ProjectController.php]: This is a part of interest.

File: resources/resources/views/pages/about/index.blade.php [hunk 4]
```diff
@@ -6,7 +6,7 @@
...
+          <p class="type--white type--marginless">We have an amazing set of faculty that pursues a diverse set of activities: different pedagogical approaches, and unique aspirations. Faculty serve the university in a variety of ways, unique to each faculty member. To more fully highlight the stories of our faculty, facilitate access to information about faculty, and facilitated collaboration among faculty and students, the Faculty App was commissioned. META+LAB, a unit within Academic Affairs, has been working on the Faculty Project, whose primary purpose is to increase the visibility of CSUN's faculty and highlight what the university represents.</p>
```
Comment (alejandrom2) [inline resources/resources/views/pages/about/index.blade.php]: LOL

File: app/Http/Controllers/PersonController.php (hunk 2 above)
Comment (arthurwirsching) [reply app/Http/Controllers/PersonController.php]: Don't remember removing this, but I will re-add it.

**Pull request merged**